iocontrol/iocontrol.go

```diff
@@ -59,10 +59,7 @@ func (io *IOControl) SetDigital(output DigitalPin, b bool) error {
 	switch pin := output.(type) {
 	case *speedgoat.DigitalPin:
 		if io.sg != nil {
-			err := io.sg.SetDigital(pin, b)
-			if err != nil {
-				return errors.Wrap(err, "set digital")
-			}
+			io.sg.SetDigital(pin, b)
```

Comment:
why drop the error checking?

Reply:
Since we're using periodic tickInputs/Outputs to read/write to the Speedgoat, the other functions will only change the pin arrays in memory and won't raise any errors. We do need to figure out better error handling for tickInputs/Outputs though